### PR TITLE
Multiple enhancements

### PR DESCRIPTION
**Pull request by torusrxxx**
NTSTATUS format function is not used in register view currently because C0000005 message is bad.

### CODE REVIEW

File: src/dbg/formatfunctions.cpp [hunk 1]
```diff
@@ -38,11 +38,53 @@ void FormatFunctions::Init()
...
+            }
+        }
+    });
+
+    Register("ntstatus", [](char* dest, size_t destCount, int argc, char* argv[], duint code, void* userdata)
+    {
+        std::vector<wchar_t> helpMessage(destCount);
+        String errName = ErrorCodeToName((unsigned int)code);
 #ifdef _WIN64
         if((code >> 32) != 0)  //Data in high part: not an error code
```
Comment (mrexodia) [inline src/dbg/formatfunctions.cpp]: The high part of NTSTATUS codes are usually set to something like 0xFFFFFFFFC0000005 so this check would say the code is invalid while it is actually valid...

File: src/gui/Src/Gui/DataCopyDialog.cpp [hunk 2]
```diff
@@ -16,23 +16,23 @@ DataCopyDialog::DataCopyDialog(const QVector<byte_t>* data, QWidget* parent) : Q
     mTypes[DataCWord] = FormatType { tr("C-Style WORD (Hex)"), 12 };
     mTypes[DataCDword] = FormatType { tr("C-Style DWORD (Hex)"), 8 };
     mTypes[DataCQword] = FormatType { tr("C-Style QWORD (Hex)"), 4 };
-    mTypes[DataCString] = FormatType { tr("C-Style String"), -1 };
```
Comment (mrexodia) [inline src/gui/Src/Gui/DataCopyDialog.cpp]: Why are these changes made?

Comment (torusrxxx) [reply src/gui/Src/Gui/DataCopyDialog.cpp]: Because the UI has changed to prevent user from setting items per line to 0.

**Comment by mrexodia** [comment]
👍

**Pull request merged**